todo_app/app.py

```diff
@@ -1,57 +1,159 @@
-import sys
+from os import error
 from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
-from todo_app.data.session_items import *
+from todo_app.trelloclient import *
+
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-#error handling for 404
+status_mapping = []
+board_Name = "ToDo"
+board_Id = ""
+
+def build_status_mapping():
+    trello_Board = TrelloBoard()
+    result = trello_Board.get_BoardList()
+    if result.status_code == 200:
+        for item in result.json():
+            if item['name'] == board_Name:
+                board_Id = item['id']   
+    else:
```

Comment:
Not that this line will only be called if the response isn't 200 - but if a board exists that isn't "ToDo" then the response will still be 200 and you'll see errors further down

Reply:
understood.  Hence why I was checking the if item['name'] == board_name, now the board_name moved to env file. Otherwise this will throw error.